tests/apps/test_common.py

```
@@ -119,11 +119,11 @@ def test_pick_manifest_from_file_empty_list(asic_gcd):
 
 
 def test_pick_manifest(asic_gcd, monkeypatch, caplog):
-    def get_manifests(pwd):
+    def get_manifests(*args):
```

Ruff (0.13.1)

Unused function argument: `args`

(ARG001)